iocage_lib/ioc_start.py

```diff
@@ -271,14 +271,12 @@ def __start_jail__(self):
             }, _callback=self.callback,
                 silent=self.silent)
 
-        if wants_dhcp:
-            self.__check_dhcp_or_accept_rtadv__(dhcp=True)
+        self.__check_dhcp_or_accept_rtadv__(True, wants_dhcp)
```

Comment:
My suggestion is to use kwargs here in case you change the interface of that function again.

Reply:
This is an internal function so even if we do change the signature later on, we should be good ? Please let me know your thoughts on this, thanks

Reply:
Well if you change the signature later, `True` could be passing True to something completely different. It's more explicit to see `ipv4=True` imo.

Reply:
Ah like that, of course we can do that. I thought you were referring to the signature, will change. Thanks